Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -39,10 +39,10 @@ fetch() {
 
   if [[ -n "$HOMEBREW_QUIET" ]]
   then
-    curl_args+=(--silent)
+    curl_args[${#curl_args[*]}]="--silent"
```

Comment:
Looked into it, and I'm not sure that's straightforward to do while keeping the quoted properties of `$HOMEBREW_USER_AGENT_CURL` unfortunately. Even when single-quoting that string within a `curl_args` string, it's interpreted as multiple arguments. Think I may leave it like this, ugly as it is.